### PR TITLE
zh-CN: Update po/zh-CN.po

### PR DESCRIPTION
**Pull request by illustager**
Update the Simplified Chinese edition of some segments: "Welcome to Comprehensive Rust", "Day 1 Morning", "Day 1 Afternoon" and "Day 2 Morning".
 
Correct some mistakes in previous translated version as well.

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -15,5 +15,5 @@ po/pt-BR.po @rastringer @hugojacob @henrif75 @joaovicmendes @jcvicelli @azevedoa
 po/ru.po @istolga @baltuky @zvonden @dyeroshenko
 po/tr.po @duyguisler @enes1313
 po/uk.po @dyeroshenko
-po/zh-CN.po @suetfei @wnghl @anlunx @kongy @noahdragon @superwhd @emmali01 @SketchK @candysonya @AgainstEntropy
+po/zh-CN.po @suetfei @wnghl @anlunx @kongy @noahdragon @superwhd @emmali01 @SketchK @candysonya @AgainstEntropy @illustager
```
Comment (kongy) [inline .github/CODEOWNERS:18]: Please move this to a separate change.

File: po/zh-CN.po [hunk 2]
```diff
@@ -1630,21 +1626,19 @@ msgstr ""
...
 msgid ""
 "The [Rust in Chromium](../chromium.md) deep dive is a half-day course on "
 "using Rust as part of the Chromium browser. It includes using Rust in "
 "Chromium's `gn` build system, bringing in third-party libraries (\"crates\") "
 "and C++ interoperability."
 msgstr ""
-"[Rust 在 Chromium 中的用法](../chromium.md) 深析是一个为期半天的课程，主要介"
-"绍了如何在 Chromium 浏览器中使用 Rust。课程内容包括如何在 Chromium 的 `gn` 构"
-"建系统中使用 Rust，引入第三方库 (\"crates\")，以及 C++ 的互操作性。"
+"[Chromium 中的 Rust深析](../chromium.md) 课程为期半天，旨在介绍如何在 "
```
Comment (kongy) [inline po/zh-CN.po:1639]: missing a space here

File: po/zh-CN.po [hunk 3]
```diff
@@ -2978,13 +2960,12 @@ msgstr ""
...
 
 #: src/types-and-values/strings.md
-#, fuzzy
 msgid "`String` - a modifiable, owned string."
 msgstr "`String` 是一个可变字符串缓冲区。"
 
 #: src/types-and-values/strings.md
 msgid "`&str` - a read-only string. String literals have this type."
-msgstr "`&str` 为只读字符串。字符串字面量属于此类型。"
+msgstr "`&str` 使只读字符串。字符串字面量属于此类型。"
```
Comment (kongy) [inline po/zh-CN.po:2968]: typo?

File: po/zh-CN.po [hunk 4]
```diff
@@ -11680,21 +11715,21 @@ msgstr ""
...
-"/// The pointers must be valid and properly aligned.\n"
+
+#: src/unsafe-rust/unsafe-functions.md src/unsafe-rust/unsafe-traits.md
+msgid "/// # Safety"
+msgstr ""
+
+#: src/unsafe-rust/unsafe-functions.md
+#, fuzzy
+msgid "/// The pointers must be valid and properly aligned."
+msgstr "在大多数情况下，指针还必须正确对齐。"
```
Comment (kongy) [inline po/zh-CN.po:11728]: "正确并对齐" should be a more accurate translation.

**Comment by mgeisler** [comment]
Hi @illustager, could you try merging the latest `main` with your branch? You'll have to resolve the conflicts and then push again.

File: .github/CODEOWNERS (hunk 1 above)
Comment (mgeisler) [reply .github/CODEOWNERS:18]: Let's just take it out here:

```suggestion
po/zh-CN.po @suetfei @wnghl @anlunx @kongy @noahdragon @superwhd @emmali01 @SketchK @candysonya @AgainstEntropy
```

**Comment by illustager** [comment]
> Hi again @illustager,
> 
> Would you be able to squash the commits on the branch into a single commit and rebase the whole thing on top of `main`? That should resolve the CLA error and also fix the merge conflict. Please make sure to apply the review comments you've already received. Thanks!

I'm so sorry for my long absence (because of heavy schoolwork). I'll finish it this night.

**Comment by mgeisler** [comment]
> I'm so sorry for my long absence (because of heavy schoolwork). I'll finish it this night.

No problem at all! Schoolwork should take priority :smile:  I was only writing to see if you were still there — people sometimes abandon PRs and then there's no need for us to keep waiting.

**Pull request closed without merging**